### ISSUE TITLE
[1.18.2] Crash on startup of modpack Create Flavored version 4.1 when adding 1.18.2-5.9.0beta2 version of Journeymap to the pack

### ISSUE THREAD

**Issue by mcdodge34** [TeamJM/journeymap#500]
Basically, I simply installed Journeymap-1.18.2-5.9.0beta2.jar to the mods folder of the modpack Create Flavored 4.1 version, and it crash the game without generating any crash reports, its very strange cause if I add the previous journeymap-1.18.2-5.9.0beta1.jar I have no crash to report, it works flawlessly. If I can be of any help, I'm willing to assist, I must specify here that I took off Xaero map mods from the mods folder of this modpack, I don't like the way the XaeroMaps work I'M an all time fan of JourneyMap, but I also tested with the 2 mods present just to make sure it wasn't the problem, but its not. I hope I made enough precisions here.

**Comment by mysticdrew** [comment]
I need logs

**Comment by mcdodge34** [comment]
I would be glad to provide them, but since there's no crash report and no crash folder, I'm not 100% sure where I could find something I can share here.

**Comment by mysticdrew** [comment]
The latest.log and journeymap.log

**Comment by mcdodge34** [comment]
Give me a minute or 2, I'll fire it up with the beta2 version once I've downloaded it again.

**Comment by mcdodge34** [comment]
Here's one of the log https://mclo.gs/frPt0ry

And let me try to locate the other hum, doesn't seem this one will be of any help, last log dates are from succesfull launch at 17h05 but I still link it here, hope it works.
[journeymap.log](https://github.com/TeamJM/journeymap/files/9570595/journeymap.log)

**Comment by mysticdrew** [comment]
That journeymap.log you provided is for beta 1

**Comment by mysticdrew** [comment]
Looks like there are mixin errors. Which did not change at all between journeymap beta1 and beta2. Looks like lithium or blueprint are the culprits. 

I only changed core journeymap code between the betas, nothing that I changed should have any issues with other mods. Something else is going on that I am not aware of.

**Comment by mcdodge34** [comment]
Yup, I know its for beta1 for the journeymap.log the beta2 doesn't even create one, it insta crash as I click on launch on PolyMC launcher. Tried launching without blueprint, doesn't launch, will try without radium to see cause I don't see any lithium, and... Strike, looks like its something about the radium mod that cause a crash, once I turn it off, it asked me to update forge, then once forge is updated, it launch without issues, now testing with Radium activated and the game launch without issue using latest forge version (40.1.80) the game had the 40.1.0 version when I installed it

**Comment by mysticdrew** [comment]
So it is working correctly now with beta2 and radium since you updated forge?

**Comment by mcdodge34** [comment]
Yes, works as intended now, we can conclude its fixed

**Comment by itsdinkd** [comment]
I get this same issue when using forge 40.1.80

**Comment by itsdinkd** [comment]
https://pastebin.com/VpdYR23y
